docs/explanation/charmed-rules.rst

```diff
@@ -0,0 +1,39 @@
+Charmed alert rules
+*******************
+
+What are alert rules?
+=====================
+
+Prometheus and Loki would post alerts to Alertmanager when alert expressions are
+evaluated as "true". Alert expressions are stored in yaml files called "alert rules".
+
+- Metrics-based alert rule expressions are written in PromQL, and are evaluated by prometheus.
+- Log-based alert rule expressions are written in LogQL, and are evaluated by Loki.
+
+As of writing, there is no official centralized store for alert rules. Often, operators
+refer to the `Awesome Prometheus Rules <https://samber.github.io/awesome-prometheus-alerts/>`_
+project as a starting point.
+
+What are charmed alert rules
```

Comment:
This section (and the one below) I think are fine for now, but they feel a bit "thin"; not sure if it's a style thing, or if we can add some extra content to it.